src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -85,8 +85,8 @@ export function transformVAOSAppointment(appt) {
     videoData: {
       isVideo,
     },
-    // communityCareProvider:
-    //   appt.start && appt.kind === 'cc' ? { id: appt.practitioners[0] } : null,
+    communityCareProvider:
```

Comment:
I would leave this commented out (or remove it). We only ever get an id from that `practitioners` array and we have to fetch the full info from PPMS like we do for requests separately.